sqrl-base/src/main/java/com/datasqrl/config/SystemBuiltInConnectors.java

```diff
@@ -0,0 +1,37 @@
+package com.datasqrl.config;
+
+import com.datasqrl.canonicalizer.Name;
+import com.datasqrl.canonicalizer.NamePath;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public
+enum SystemBuiltInConnectors {
+
+  PRINT_SINK(Name.system("print"), ExternalDataType.sink),
+  LOCAL_FILE_SOURCE(Name.system("localfile"), ExternalDataType.source),
+  LOG(Name.system("log"), ExternalDataType.sink);
```

Comment:
im not sure why we need this... We need to book keep very little when a 'log' export is done.

Reply:
This section was added by @mbroecheler, so he would be the best person to explain its purpose. From my understanding, it aids comprehension. It seems unusual that we check whether we can create a ConnectorFactory in the ImportExportAnalyzer without using it. Later, in the ScriptPlanner, we create the same ConnectorFactory and then use it. I believe it's cleaner to check here if we have a built-in connector and then later create the corresponding connector definition for it.

Reply:
These are the connectors that are "hard-coded" into SQRL and always available. They are instantiated based on configuration in the package.json.
The goal was to have these centrally defined so that we do not have string constants floating all over the codebase. This makes it easier to understand what the pre-defined connectors are, what their name is, and whether they are sources or sinks.
I added this mostly to refactor what we had before because I noticed that we had a proliferation of string constants.